code/__DEFINES/xenoartifact_materials.dm

```diff
@@ -0,0 +1,19 @@
+//Xenoartifact defines
+//Material defines. Used for characteristic generation.
+#define BLUESPACE "#1e7cff" //Silly toys
+#define PLASMA "#ff00c8" //Associated weapons
+#define URANIUM "#00ff0d" //Broken Enigmas
+#define BANANIUM "#ffd900" //Wildcard, may god have mercy. Was originally called BANANIUM, changed for localisation.
+#define DEBUGIUM "#ff4800"
+
+//Technically not materials, still used for xenoartifact related content
+#define XENO_SELLER_NAMES list("Borov", "Ivantsov", "Petrenko", "Voronin", "Kitsenko", "Plichko", "Sergei", "Kruglov", "Sakharov", "Kalugin", "Semenov", "Vasiliev", "Pavlik", "Tolik", "Kuznetsov", "Sidorovich", "Strelok")
+#define XENO_SELLER_DIAL list("Hello, Commrade. I think I have something that might interest you.","Hello, Friend. I think I have something you might be interested in.","Commrade, I can offer you only this.","For you, my Friend, I offer this.", "Commrade, this thing killed my Babushka, take it.","Friend, you want?","My buddy thinks I could sell this.","I'm pretty sure this took several years off my life, take it.","This was hard to find, but you can have it.","I found this one deep in the zone, it was a risk to get.")
```

Comment:
How often are they used? Do we really need to define list declaration?

Reply:
I think the Corgi might've told me to.